src/components/customPageComponents/SubpageVideoHeroSection.tsx

```diff
@@ -12,16 +12,27 @@ interface SubpageVideoHeroSectionProps {
 const SubpageVideoHeroSection = ({ children }: SubpageVideoHeroSectionProps) => {
   const innerRef = useRef<HTMLElement>(null)
   const videoRef = useRef<HTMLVideoElement>(null)
+  const scheduledRef = useRef(false)
+  const lastUpdateRef = useRef(0)
 
   const handleMouseMove = (e: React.MouseEvent<HTMLElement>) => {
-    if (videoRef.current) {
-      const bounds = e.currentTarget.getBoundingClientRect()
-      const relativeX = e.clientX - bounds.left
-      const ratio = relativeX / bounds.width
-      const duration = videoRef.current.duration
-      if (duration) {
-        videoRef.current.currentTime = ratio * duration
-      }
+    const now = performance.now()
+    if (now - lastUpdateRef.current < 100) return
```

Comment:
16 would simulate 60fps and 33 would simulate 30fps. I tried that too but without a result that worked on Chrome in a satisfying way. 100 seems to be the compromise?